chromium_src/components/content_settings/core/common/content_settings_types.mojom

```diff
@@ -24,6 +24,7 @@ enum ContentSettingsType {
   BRAVE_HTTPS_UPGRADE,
   BRAVE_REMEMBER_1P_STORAGE,
   BRAVE_LOCALHOST_ACCESS,
+  BRAVE_AI_CHAT,
```

Comment:
it's not clear what exactly this allows a site to do. Can you provide more info here on the details?

Reply:
If user has this permission set to allowed for Brave Search (for example, accept the permission prompt before), when click on the specific button URL https://search.brave.com/leo#some_nonce, Leo side panel will be opened.

Reply:
Sorry, I meant add some comments to explain, but maybe it should be `BRAVE_OPEN_AI_CHAT`?

Reply:
ah okay, I added a comment in https://github.com/brave/brave-core/pull/26082/commits/0db3a85a4590c6a71f8fbaaf310370de6e5aff3e.
I was thinking of it more like accessing AI Chat (via opening it), and seems these types rarely have a verb before the resource, that's why I renamed it from `BRAVE_OPEN_AI_CHAT` to `BRAVE_AI_CHAT` during my development.
Is keeping the name with added comments okay?
Let me know if you have a strong preference to change the names here (and also permission context associated to it?).
Thanks.

Reply:
They seem to have verbs: "_upgrade_ https", "_remember_ 1p storage", "_access_ localhost". Do we think we'd lump access to the actual conversation data in this same setting type?

Reply:
I do think it would be clearer to understand if it was something like `BRAVE_CONTINUE_IN_AI_CHAT` as it's not "enabling AI Chat" for that site.

Reply:
it's now BRAVE_OPEN_AI_CHAT